demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py

```diff
@@ -124,6 +124,29 @@ def validate_toversion(
     }
 
 
+def get_items_used_deprecated(tx: Transaction, file_paths: List[str]):
+    files_filter = (
+        f"AND (p.path in {file_paths} OR d.path in {file_paths})" if file_paths else ""
+    )
+    command_query = f"""// Returning all the items which using deprecated commands
+    match (p{{deprecated: false}})-[:USES]->(c:Command)<-[:HAS_COMMAND{{deprecated: true}}]-(d:Integration) where not p.is_test
+optional match (i2:Integration)-[:HAS_COMMAND{{deprecated: false}}]->(c)
+with p, c, i2
+where i2 is null
+{files_filter}
+return c.object_id as deprecated_command, collect(p.path) as object_using_deprecated
+"""
```

Comment:
```suggestion
    command_query = f"""// Returning all the items which using deprecated commands
MATCH (p{{deprecated: false}})-[:USES]->(c:Command)<-[:HAS_COMMAND{{deprecated: true}}]-(i:Integration) WHERE NOT p.is_test
OPTIONAL MATCH (i2:Integration)-[:HAS_COMMAND{{deprecated: false}}]->(c)
WHERE id(i) <> id(i2)
WITH p, c, i2
WHERE i2 IS NULL
{files_filter}
RETURN c.object_id AS deprecated_command, collect(p.path) AS object_using_deprecated"""
```
1. uppercase neo4j keywords are a convention
2. note the added condition: `WHERE id(i) <> id(i2)` - to ensure there is no available replacement for the deprecated command